CHANGELOG.md

```diff
@@ -4,19 +4,19 @@ All notable changes to the Imperative package will be documented in this file.
 
 ## `4.8.1`
 
-- Updated the SSO Callback so it doesn't prompt for user/password if a token exists
+- Fixed an issue with `ConnectionPropsForSessCfg` where the user would be prompted for user/password even if a token was present. [#436](https://github.com/zowe/imperative/pull/436)
 
 ## `4.8.0`
 
-- Add the SSO Callback to allow other applications to call their own function in validating sessions
+- Added the SSO Callback funtion, which allows applications to call their own functions in validating sessions. The callback option is named `getValuesBack`. [#422](https://github.com/zowe/imperative/issues/422)
```

Comment:
What is meant by "...in validating sessions."?

Reply:
I think it means "while validating session properties", such as host, port, user, password, token, etc.